Cargo.toml

```diff
@@ -34,6 +34,7 @@ glob = "0.3.0"
 hex = "0.4"
 home = "0.5"
 humantime = "2.0.0"
+humansize = "2.1.0"
```

Comment:
We already have an in-house implementation for this. Should we share this snippet without introducing a new dependency?

https://github.com/rust-lang/cargo/blob/23ab870c50b13dba93b8ef0d9d69c29521610805/src/cargo/sources/git/utils.rs#L758-L762

Reply:
I moved `format_bytes` to `cargo/src/cargo/util/mod.rs` so it could be shared. Since `format_bytes` uses binary rather than decimal byte units, the output format is now like `Packaged 1160 files, 7.9MiB (2.0MiB compressed)`.